src/Core/Extensible.php

```diff
@@ -18,7 +17,18 @@
  */
 trait Extensible
 {
-    use CustomMethods;
+    use CustomMethods {
```

Comment:
Yeah — what did you think about including a "warmPropertyMethod" option that can be set to "getExtensionInstances" on an Extensible object?